embassy-stm32-wpan/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "embassy-stm32-wpan"
-version = "0.1001.1"
+version = "0.1002.0"
```

Comment:
patch update instead of minor update?

Reply:
Adds a new command.

"MINOR version when you add functionality in a backward compatible manner"

Reply:
oh, cool. didn't realize it did that. i thought it just _utilized_ a new feature in hci, not exposed that feature out through wpan :+1: